junit-runtime/src/main/scala/org/scalajs/junit/JUnitMasterRunner.scala

```diff
@@ -0,0 +1,82 @@
+package org.scalajs.junit
+
+import com.novocode.junit.RunSettings
+import sbt.testing._
+import java.util.concurrent.atomic.AtomicInteger
+
+final class JUnitMasterRunner(
+    args: Array[String],
+    remoteArgs: Array[String],
+    testClassLoader: ClassLoader,
+    runSettings: RunSettings)
+    extends JUnitBaseRunner(args, remoteArgs, testClassLoader, runSettings) {
+
+  private[this] val registeredCount = new AtomicInteger(0)
```

Comment:
Do we really want atomics, here?
For cross-compiling testing frameworks, they are necessary because there can be multiple threads. But this testing framework is Scala.js only, so it doesn't need that.